hack/.ci/run-e2e-gke.sh

```diff
@@ -36,6 +36,7 @@ if [ -z ${ARTIFACTS+x} ]; then
   exit 2
 fi
 
+SO_NODECONFIG_PATH=${SO_NODECONFIG_PATH=./hack/.ci/manifests/cluster/nodeconfig.yaml}
```

Comment:
this seems to work but elsewhere we use `:-` - what's the difference?

Reply:
`:-` sets variable to default value when variable is explicitly set to empty string, where `=` keep it as is.

Reply:
thx